### DISCUSSION TITLE
I can't read the name of my customers

### DISCUSSION THREAD

**Discussion by luisdanielhj** [chatwoot/chatwoot#5889] [Self hosted] [answered]
Hi, I just installed chatwoot self hosted and I made my application in meta, ask for the permissions and all is working, but I can't read the name of my customers, every customer who send me a message shows their name as John Doe, if me or other people admins of the Facebook page send a message I can see their names.

Probably I forgot to ask some permission to meta, can you help me guys in order to know what else do I need to get my app working? 

Thank You!

![Screenshot 2022-11-17 at 18 21 07](https://user-images.githubusercontent.com/106158566/202588017-a25a7dd2-9e0b-4167-82f6-a95c769225df.png)

**Comment by sojan-official** [comment, marked as answer]
ref: https://github.com/chatwoot/chatwoot/issues/5567#issuecomment-1270277835

**Reply by luisdanielhj** [reply]
what is the exact name of the permission who allows me to read my customers name?

**Reply by sojan-official** [reply]
ref: https://developers.facebook.com/docs/permissions/reference/pages_read_engagement
